res/controllers/Pioneer DDJ-200.midi.xml

```diff
@@ -3,7 +3,7 @@
     <info>
         <name>Pioneer DDJ-200</name>
         <author>Dan Giddins, Frank.Breitling at gmx.de</author>
-        <description>2-deck USB and Bluetooth MIDI controller.</description>
+        <description>2-deck USB and Bluetooth MIDI controller supporting 4 decks.</description>
```

Comment:
Maybe this? Otherwise the sentence sounds like a paradox.

```suggestion
        <description>2-deck USB and Bluetooth MIDI controller with support for 4-deck mode.</description>
```